Yep no entity_id anymore

<img width="1292" alt="image" src="https://github.com/metabase/metabase/assets/110378427/a530cd19-b4bb-4b14-9fc4-ca997b9bfb5d">

BUT there is an issue cause no serialization just doesn't work:

<img width="1512" alt="image" src="https://github.com/metabase/metabase/assets/110378427/23e705b4-f3e4-4bb9-8da2-53129bb6b9f5">

Just says 

```
2023-10-27 15:17:27,431 INFO db.data-migrations :: Running all necessary data migrations, this may take a minute.
clojure.lang.ExceptionInfo: Table "DATA_MIGRATIONS" not found; SQL statement:
SELECT * FROM "DATA_MIGRATIONS" [42102-214] {:toucan2/context-trace [["execute SQL with class com.mchange.v2.c3p0.impl.NewProxyConnection" {:toucan2.jdbc.query/sql-args ["SELECT * FROM \"DATA_MIGRATIONS\""]}] ["resolve connection" {:toucan2.connection/connectable metabase.db.connection.ApplicationDB}] ["resolve connection" {:toucan2.connection/connectable :default}] ["resolve connection" {:toucan2.connection/connectable nil}] {:toucan2.pipeline/rf #object[clojure.core$map$fn__5931$fn__5932 0x6451cdcb "clojure.core$map$fn__5931$fn__5932@6451cdcb"]} ["with compiled query" {:toucan2.pipeline/compiled-query ["SELECT * FROM \"DATA_MIGRATIONS\""]}] ["with built query" {:toucan2.pipeline/built-query {:select [:*], :from [[:data_migrations]]}}] ["with resolved query" {:toucan2.pipeline/resolved-query {}}] ["with parsed args" {:toucan2.pipeline/query-type :toucan.query-type/select.instances.fns, :toucan2.pipeline/parsed-args {:queryable {}}}] ["with model" {:toucan2.pipeline/model :model/DataMigrations}] ["with unparsed args" {:toucan2.pipeline/query-type :toucan.query-type/select.instances.fns, :toucan2.pipeline/unparsed-args (:model/DataMigrations)}]]}
        at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
        at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
        at org.h2.message.DbException.get(DbException.java:223)
        at org.h2.message.DbException.get(DbException.java:199)
        at org.h2.command.Parser.getTableOrViewNotFoundDbException(Parser.java:8398)
        at org.h2.command.Parser.getTableOrViewNotFoundDbException(Parser.java:8369)
        at org.h2.command.Parser.readTableOrView(Parser.java:8358)
        at org.h2.command.Parser.readTablePrimary(Parser.java:1863)
        at org.h2.command.Parser.readTableReference(Parser.java:2334)
        at org.h2.command.Parser.parseSelectFromPart(Parser.java:2772)
        at org.h2.command.Parser.parseSelect(Parser.java:2878)
        at org.h2.command.Parser.parseQueryPrimary(Parser.java:2762)
        at org.h2.command.Parser.parseQueryTerm(Parser.java:2633)
        at org.h2.command.Parser.parseQueryExpressionBody(Parser.java:2612)
        at org.h2.command.Parser.parseQueryExpressionBodyAndEndOfQuery(Parser.java:2605)
        at org.h2.command.Parser.parseQueryExpression(Parser.java:2598)
        at org.h2.command.Parser.parseQuery(Parser.java:2567)
        at org.h2.command.Parser.parsePrepared(Parser.java:724)
        at org.h2.command.Parser.parse(Parser.java:689)
        at org.h2.command.Parser.parse(Parser.java:661)
        at org.h2.command.Parser.prepareCommand(Parser.java:569)
        at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
        at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
        at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
        at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:92)
        at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:1015)
        at com.mchange.v2.c3p0.impl.NewProxyConnection.prepareStatement(NewProxyConnection.java:509)
        at next.jdbc.prepare$create.invokeStatic(prepare.clj:121)
        at next.jdbc.prepare$create.invoke(prepare.clj:84)
        at next.jdbc.prepare$fn__26623.invokeStatic(prepare.clj:150)
        at next.jdbc.prepare$fn__26623.invoke(prepare.clj:147)
        at next.jdbc.protocols$fn__26367$G__26362__26376.invoke(protocols.clj:50)
        at next.jdbc$prepare.invokeStatic(jdbc.clj:196)
        at next.jdbc$prepare.invoke(jdbc.clj:178)
        at toucan2.jdbc.query$reduce_jdbc_query.invokeStatic(query.clj:33)
        at toucan2.jdbc.query$reduce_jdbc_query.invoke(query.clj:22)
        at toucan2.query_execution_backend.jdbc$transduce_execute_with_connection_primary_method_java_sql_Connection_default_default.invokeStatic(jdbc.clj:21)
        at toucan2.query_execution_backend.jdbc$transduce_execute_with_connection_primary_method_java_sql_Connection_default_default.invoke(jdbc.clj:11)
        at clojure.lang.AFn.applyToHelper(AFn.java:178)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.core$apply.invokeStatic(core.clj:675)
        at clojure.core$partial$fn__5908.doInvoke(core.clj:2639)
        at clojure.lang.RestFn.applyTo(RestFn.java:146)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.core$apply.invokeStatic(core.clj:667)
        at clojure.core$apply.invoke(core.clj:662)
        at methodical.impl.combo.threaded$fn__23398$fn__23399$fn__23406.invoke(threaded.clj:79)
        at methodical.impl.combo.threaded$reducer_fn$fn__23368$fn__23372.invoke(threaded.clj:23)
        at clojure.lang.ArrayChunk.reduce(ArrayChunk.java:58)
        at clojure.core.protocols$fn__8244.invokeStatic(protocols.clj:136)
        at clojure.core.protocols$fn__8244.invoke(protocols.clj:124)
        at clojure.core.protocols$fn__8204$G__8199__8213.invoke(protocols.clj:19)
        at clojure.core.protocols$seq_reduce.invokeStatic(protocols.clj:31)
        at clojure.core.protocols$fn__8236.invokeStatic(protocols.clj:75)
        at clojure.core.protocols$fn__8236.invoke(protocols.clj:75)
        at clojure.core.protocols$fn__8178$G__8173__8191.invoke(protocols.clj:13)
        at clojure.core$reduce.invokeStatic(core.clj:6886)
        at clojure.core$reduce.invoke(core.clj:6868)
        at methodical.impl.combo.threaded$reducer_fn$fn__23368.invoke(threaded.clj:21)
        at clojure.core$comp$fn__5876.doInvoke(core.clj:2589)
        at clojure.lang.RestFn.applyTo(RestFn.java:146)
        at clojure.core$apply.invokeStatic(core.clj:675)
        at clojure.core$apply.doInvoke(core.clj:662)
        at clojure.lang.RestFn.invoke(RestFn.java:533)
        at methodical.impl.combo.threaded$combine_with_threader$fn__23378.doInvoke(threaded.clj:46)
        at clojure.lang.RestFn.applyTo(RestFn.java:151)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.core$apply.invokeStatic(core.clj:675)
        at clojure.core$apply.doInvoke(core.clj:662)
        at clojure.lang.RestFn.invoke(RestFn.java:533)
        at methodical.impl.standard$invoke_multifn.invokeStatic(standard.clj:65)
        at methodical.impl.standard$invoke_multifn.doInvoke(standard.clj:47)
        at clojure.lang.RestFn.invoke(RestFn.java:594)
        at methodical.impl.standard.StandardMultiFn.invoke(standard.clj:199)
        at toucan2.pipeline$transduce_execute$with_connection_STAR___28126.invoke(pipeline.clj:86)
        at toucan2.connection$bind_current_connectable_fn$fn__27739.invoke(connection.clj:104)
        at toucan2.connection$bind_current_connectable_fn$fn__27739.invoke(connection.clj:104)
        at toucan2.connection$bind_current_connectable_fn$fn__27739.invoke(connection.clj:104)
        at toucan2.connection$do_with_connection_primary_method_javax_sql_DataSource.invokeStatic(connection.clj:213)
        at toucan2.connection$do_with_connection_primary_method_javax_sql_DataSource.invoke(connection.clj:210)
        at clojure.lang.AFn.applyToHelper(AFn.java:160)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:436)
        at clojure.core$partial$fn__5908.invoke(core.clj:2642)
        at clojure.lang.AFn.applyToHelper(AFn.java:156)
        at clojure.lang.RestFn.applyTo(RestFn.java:132)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at methodical.impl.combo.threaded$fn__23398$fn__23399$fn__23400.invoke(threaded.clj:70)
        at methodical.impl.combo.threaded$reducer_fn$fn__23368$fn__23372.invoke(threaded.clj:23)
        at clojure.lang.ArrayChunk.reduce(ArrayChunk.java:58)
        at clojure.core.protocols$fn__8244.invokeStatic(protocols.clj:136)
        at clojure.core.protocols$fn__8244.invoke(protocols.clj:124)
        at clojure.core.protocols$fn__8204$G__8199__8213.invoke(protocols.clj:19)
        at clojure.core.protocols$seq_reduce.invokeStatic(protocols.clj:31)
        at clojure.core.protocols$fn__8236.invokeStatic(protocols.clj:75)
        at clojure.core.protocols$fn__8236.invoke(protocols.clj:75)
        at clojure.core.protocols$fn__8178$G__8173__8191.invoke(protocols.clj:13)
        at clojure.core$reduce.invokeStatic(core.clj:6886)
        at clojure.core$reduce.invoke(core.clj:6868)
        at methodical.impl.combo.threaded$reducer_fn$fn__23368.invoke(threaded.clj:21)
        at clojure.core$comp$fn__5876.invoke(core.clj:2587)
        at methodical.impl.combo.threaded$combine_with_threader$fn__23378.invoke(threaded.clj:43)
        at clojure.lang.AFn.applyToHelper(AFn.java:156)
        at clojure.lang.RestFn.applyTo(RestFn.java:132)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at toucan2.connection$do_with_connection_around_method_toucan2_connection_default.invokeStatic(connection.clj:118)
        at toucan2.connection$do_with_connection_around_method_toucan2_connection_default.invoke(connection.clj:106)
        at clojure.lang.AFn.applyToHelper(AFn.java:160)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:436)
        at clojure.core$partial$fn__5908.invoke(core.clj:2642)
        at clojure.lang.AFn.applyToHelper(AFn.java:156)
        at clojure.lang.RestFn.applyTo(RestFn.java:132)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at methodical.impl.standard$invoke_multifn.invokeStatic(standard.clj:55)
        at methodical.impl.standard$invoke_multifn.invoke(standard.clj:47)
        at methodical.impl.standard.StandardMultiFn.invoke(standard.clj:193)
        at metabase.db.connection$do_with_connection_primary_method_default.invokeStatic(connection.clj:139)
        at metabase.db.connection$do_with_connection_primary_method_default.invoke(connection.clj:137)
        at clojure.lang.AFn.applyToHelper(AFn.java:160)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:436)
        at clojure.core$partial$fn__5908.invoke(core.clj:2642)
        at clojure.lang.AFn.applyToHelper(AFn.java:156)
        at clojure.lang.RestFn.applyTo(RestFn.java:132)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at methodical.impl.combo.threaded$fn__23398$fn__23399$fn__23400.invoke(threaded.clj:70)
        at methodical.impl.combo.threaded$reducer_fn$fn__23368$fn__23372.invoke(threaded.clj:23)
        at clojure.lang.ArrayChunk.reduce(ArrayChunk.java:58)
        at clojure.core.protocols$fn__8244.invokeStatic(protocols.clj:136)
        at clojure.core.protocols$fn__8244.invoke(protocols.clj:124)
        at clojure.core.protocols$fn__8204$G__8199__8213.invoke(protocols.clj:19)
        at clojure.core.protocols$seq_reduce.invokeStatic(protocols.clj:31)
        at clojure.core.protocols$fn__8236.invokeStatic(protocols.clj:75)
        at clojure.core.protocols$fn__8236.invoke(protocols.clj:75)
        at clojure.core.protocols$fn__8178$G__8173__8191.invoke(protocols.clj:13)
        at clojure.core$reduce.invokeStatic(core.clj:6886)
        at clojure.core$reduce.invoke(core.clj:6868)
        at methodical.impl.combo.threaded$reducer_fn$fn__23368.invoke(threaded.clj:21)
        at clojure.core$comp$fn__5876.invoke(core.clj:2587)
        at methodical.impl.combo.threaded$combine_with_threader$fn__23378.invoke(threaded.clj:43)
        at clojure.lang.AFn.applyToHelper(AFn.java:156)
        at clojure.lang.RestFn.applyTo(RestFn.java:132)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at toucan2.connection$do_with_connection_around_method_toucan2_connection_default.invokeStatic(connection.clj:118)
        at toucan2.connection$do_with_connection_around_method_toucan2_connection_default.invoke(connection.clj:106)
        at clojure.lang.AFn.applyToHelper(AFn.java:160)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:436)
        at clojure.core$partial$fn__5908.invoke(core.clj:2642)
        at clojure.lang.AFn.applyToHelper(AFn.java:156)
        at clojure.lang.RestFn.applyTo(RestFn.java:132)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at methodical.impl.standard$invoke_multifn.invokeStatic(standard.clj:55)
        at methodical.impl.standard$invoke_multifn.invoke(standard.clj:47)
        at methodical.impl.standard.StandardMultiFn.invoke(standard.clj:193)
        at toucan2.connection$do_with_connection_primary_method_.invokeStatic(connection.clj:204)
        at toucan2.connection$do_with_connection_primary_method_.invoke(connection.clj:194)
        at clojure.lang.AFn.applyToHelper(AFn.java:160)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:436)
        at clojure.core$partial$fn__5908.invoke(core.clj:2642)
        at clojure.lang.AFn.applyToHelper(AFn.java:156)
        at clojure.lang.RestFn.applyTo(RestFn.java:132)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at methodical.impl.combo.threaded$fn__23398$fn__23399$fn__23400.invoke(threaded.clj:70)
        at methodical.impl.combo.threaded$reducer_fn$fn__23368$fn__23372.invoke(threaded.clj:23)
        at clojure.lang.ArrayChunk.reduce(ArrayChunk.java:58)
        at clojure.core.protocols$fn__8244.invokeStatic(protocols.clj:136)
        at clojure.core.protocols$fn__8244.invoke(protocols.clj:124)
        at clojure.core.protocols$fn__8204$G__8199__8213.invoke(protocols.clj:19)
        at clojure.core.protocols$seq_reduce.invokeStatic(protocols.clj:31)
        at clojure.core.protocols$fn__8236.invokeStatic(protocols.clj:75)
        at clojure.core.protocols$fn__8236.invoke(protocols.clj:75)
        at clojure.core.protocols$fn__8178$G__8173__8191.invoke(protocols.clj:13)
        at clojure.core$reduce.invokeStatic(core.clj:6886)
        at clojure.core$reduce.invoke(core.clj:6868)
        at methodical.impl.combo.threaded$reducer_fn$fn__23368.invoke(threaded.clj:21)
        at clojure.core$comp$fn__5876.invoke(core.clj:2587)
        at methodical.impl.combo.threaded$combine_with_threader$fn__23378.invoke(threaded.clj:43)
        at clojure.lang.AFn.applyToHelper(AFn.java:156)
        at clojure.lang.RestFn.applyTo(RestFn.java:132)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at toucan2.connection$do_with_connection_around_method_toucan2_connection_default.invokeStatic(connection.clj:118)
        at toucan2.connection$do_with_connection_around_method_toucan2_connection_default.invoke(connection.clj:106)
        at clojure.lang.AFn.applyToHelper(AFn.java:160)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:436)
        at clojure.core$partial$fn__5908.invoke(core.clj:2642)
        at clojure.lang.AFn.applyToHelper(AFn.java:156)
        at clojure.lang.RestFn.applyTo(RestFn.java:132)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at methodical.impl.standard$invoke_multifn.invokeStatic(standard.clj:55)
        at methodical.impl.standard$invoke_multifn.invoke(standard.clj:47)
        at methodical.impl.standard.StandardMultiFn.invoke(standard.clj:193)
        at toucan2.pipeline$transduce_execute.invokeStatic(pipeline.clj:84)
        at toucan2.pipeline$transduce_execute.invoke(pipeline.clj:70)
        at clojure.lang.Var.invoke(Var.java:399)
        at toucan2.pipeline$transduce_compiled_query.invokeStatic(pipeline.clj:278)
        at toucan2.pipeline$transduce_compiled_query.invoke(pipeline.clj:274)
        at toucan2.pipeline$transduce_built_query.invokeStatic(pipeline.clj:286)
        at toucan2.pipeline$transduce_built_query.invoke(pipeline.clj:280)
        at toucan2.pipeline$transduce_query_primary_method_default.invokeStatic(pipeline.clj:312)
        at toucan2.pipeline$transduce_query_primary_method_default.invoke(pipeline.clj:309)
        at clojure.lang.AFn.applyToHelper(AFn.java:178)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.core$apply.invokeStatic(core.clj:675)
        at clojure.core$partial$fn__5908.doInvoke(core.clj:2639)
        at clojure.lang.RestFn.applyTo(RestFn.java:146)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.core$apply.invokeStatic(core.clj:667)
        at clojure.core$apply.invoke(core.clj:662)
        at methodical.impl.combo.threaded$fn__23398$fn__23399$fn__23406.invoke(threaded.clj:79)
        at methodical.impl.combo.threaded$reducer_fn$fn__23368$fn__23372.invoke(threaded.clj:23)
        at clojure.lang.ArrayChunk.reduce(ArrayChunk.java:58)
        at clojure.core.protocols$fn__8244.invokeStatic(protocols.clj:136)
        at clojure.core.protocols$fn__8244.invoke(protocols.clj:124)
        at clojure.core.protocols$fn__8204$G__8199__8213.invoke(protocols.clj:19)
        at clojure.core.protocols$seq_reduce.invokeStatic(protocols.clj:31)
        at clojure.core.protocols$fn__8236.invokeStatic(protocols.clj:75)
        at clojure.core.protocols$fn__8236.invoke(protocols.clj:75)
        at clojure.core.protocols$fn__8178$G__8173__8191.invoke(protocols.clj:13)
        at clojure.core$reduce.invokeStatic(core.clj:6886)
        at clojure.core$reduce.invoke(core.clj:6868)
        at methodical.impl.combo.threaded$reducer_fn$fn__23368.invoke(threaded.clj:21)
        at clojure.core$comp$fn__5876.doInvoke(core.clj:2589)
        at clojure.lang.RestFn.applyTo(RestFn.java:146)
        at clojure.core$apply.invokeStatic(core.clj:675)
        at clojure.core$apply.doInvoke(core.clj:662)
        at clojure.lang.RestFn.invoke(RestFn.java:533)
        at methodical.impl.combo.threaded$combine_with_threader$fn__23378.doInvoke(threaded.clj:46)
        at clojure.lang.RestFn.applyTo(RestFn.java:151)
        at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.core$apply.invokeStatic(core.clj:675)
        at clojure.core$apply.doInvoke(core.clj:662)
        at clojure.lang.RestFn.invoke(RestFn.java:533)
        at methodical.impl.standard$invoke_multifn.invokeStatic(standard.clj:65)
        at methodical.impl.standard$invoke_multifn.doInvoke(standard.clj:47)
        at clojure.lang.RestFn.invoke(RestFn.java:594)
        at methodical.impl.standard.StandardMultiFn.invoke(standard.clj:199)
        at toucan2.pipeline$transduce_query_STAR_.invokeStatic(pipeline.clj:318)
        at toucan2.pipeline$transduce_query_STAR_.invoke(pipeline.clj:314)
        at toucan2.pipeline$transduce_with_model.invokeStatic(pipeline.clj:333)
        at toucan2.pipeline$transduce_with_model.invoke(pipeline.clj:320)
        at toucan2.pipeline$transduce_parsed.invokeStatic(pipeline.clj:349)
        at toucan2.pipeline$transduce_parsed.invoke(pipeline.clj:335)
        at toucan2.pipeline$transduce_unparsed.invokeStatic(pipeline.clj:357)
        at toucan2.pipeline$transduce_unparsed.invoke(pipeline.clj:351)
        at toucan2.select$select_fn_set.invokeStatic(select.clj:81)
        at toucan2.select$select_fn_set.doInvoke(select.clj:68)
        at clojure.lang.RestFn.applyTo(RestFn.java:139)
        at clojure.core$apply.invokeStatic(core.clj:671)
        at clojure.core$apply.invoke(core.clj:662)
        at toucan2.select$select_pks_set.invokeStatic(select.clj:143)
        at toucan2.select$select_pks_set.doInvoke(select.clj:130)
        at clojure.lang.RestFn.invoke(RestFn.java:410)
        at metabase.db.data_migrations$run_all_BANG_.invokeStatic(data_migrations.clj:70)
        at metabase.db.data_migrations$run_all_BANG_.invoke(data_migrations.clj:66)
        at clojure.lang.Var.invoke(Var.java:380)
        at metabase.db.setup$fn__52140$run_data_migrations_BANG___52145$fn__52146.invoke(setup.clj:140)
        at metabase.db.setup$fn__52140$run_data_migrations_BANG___52145.invoke(setup.clj:134)
        at metabase.db.setup$fn__52166$setup_db_BANG___52171$fn__52172$fn__52175$fn__52176.invoke(setup.clj:157)
        at metabase.util.jvm$do_with_us_locale.invokeStatic(jvm.clj:239)
        at metabase.util.jvm$do_with_us_locale.invoke(jvm.clj:225)
        at metabase.db.setup$fn__52166$setup_db_BANG___52171$fn__52172$fn__52175.invoke(setup.clj:152)
        at metabase.db.setup$fn__52166$setup_db_BANG___52171$fn__52172.invoke(setup.clj:151)
        at metabase.db.setup$fn__52166$setup_db_BANG___52171.invoke(setup.clj:145)
        at metabase.db$setup_db_BANG_$fn__52199.invoke(db.clj:69)
        at metabase.db$setup_db_BANG_.invokeStatic(db.clj:64)
        at metabase.db$setup_db_BANG_.invoke(db.clj:55)
        at metabase_enterprise.serialization.cmd$v2_dump.invokeStatic(cmd.clj:207)
        at metabase_enterprise.serialization.cmd$v2_dump.invoke(cmd.clj:203)
        at clojure.lang.AFn.applyToHelper(AFn.java:156)
        at clojure.lang.AFn.applyTo(AFn.java:144)
        at clojure.lang.Var.applyTo(Var.java:705)
        at clojure.core$apply.invokeStatic(core.clj:667)
        at clojure.core$apply.invoke(core.clj:662)
        at metabase.cmd$call_enterprise.invokeStatic(cmd.clj:53)
        at metabase.cmd$call_enterprise.doInvoke(cmd.clj:43)
        at clojure.lang.RestFn.invoke(RestFn.java:439)
        at metabase.cmd$export.invokeStatic(cmd.clj:217)
        at metabase.cmd$export.doInvoke(cmd.clj:206)
        at clojure.lang.RestFn.invoke(RestFn.java:410)
        at clojure.lang.AFn.applyToHelper(AFn.java:154)
        at clojure.lang.RestFn.applyTo(RestFn.java:132)
        at clojure.core$apply.invokeStatic(core.clj:667)
        at clojure.core$apply.invoke(core.clj:662)
        at metabase.cmd$run_cmd$fn__118488.invoke(cmd.clj:293)
        at metabase.cmd$run_cmd.invokeStatic(cmd.clj:292)
        at metabase.cmd$run_cmd.invoke(cmd.clj:282)
        at clojure.lang.Var.invoke(Var.java:388)
        at metabase.core$run_cmd.invokeStatic(core.clj:181)
        at metabase.core$run_cmd.invoke(core.clj:179)
        at metabase.core$_main.invokeStatic(core.clj:203)
        at metabase.core$_main.doInvoke(core.clj:198)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.lang.Var.applyTo(Var.java:705)
        at clojure.core$apply.invokeStatic(core.clj:667)
        at clojure.core$apply.invoke(core.clj:662)
        at metabase.bootstrap$_main.invokeStatic(bootstrap.clj:31)
        at metabase.bootstrap$_main.doInvoke(bootstrap.clj:28)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at metabase.bootstrap.main(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "DATA_MIGRATIONS" not found; SQL statement:
SELECT * FROM "DATA_MIGRATIONS" [42102-214]
        ... 326 more
Command failed with exception: Table "DATA_MIGRATIONS" not found; SQL statement:
SELECT * FROM "DATA_MIGRATIONS" [42102-214]
```

Did you get the same issue when running ` java -jar 1.47.4.jar export export_with_no_entity_ids`  @qnkhuat

--- a/enterprise/backend/test/metabase_enterprise/serialization/v2/entity_ids_test.clj
+++ b/enterprise/backend/test/metabase_enterprise/serialization/v2/entity_ids_test.clj
@@ -1,5 +1,6 @@
 (ns metabase-enterprise.serialization.v2.entity-ids-test
   (:require
+   [clojure.set :as set]
    [clojure.string :as str]
    [clojure.test :refer :all]
    [metabase-enterprise.serialization.v2.entity-ids :as v2.entity-ids]
@@ -66,3 +67,50 @@
           (is (= true
                  (v2.entity-ids/drop-entity-ids!)))
           (is (nil? (t2/select-fn-set :entity-id Dashboard))))))))
+
+(deftest entity-models-test
+  (testing "Sanity check: list of models that does not need an entity_id column,
+           if this test fails, check if it really needs it.
+           If yes, makes surethe exported data includes `:entity_id` column
+           or the model implements [[serdes/hash-fields]] (#35097)"
+    (is (= #{:model/MetricImportantField
+             :model/ModerationReview
+             :model/CollectionBookmark
+             :model/Secret
+             :model/GroupTableAccessPolicy
+             :model/FieldValues
+             :model/ModelIndex
+             :model/DashboardCardSeries
+             :model/ParameterCard
+             :model/QueryAction
+             :model/ImplicitAction
+             :model/User
+             :model/Revision
+             :model/PermissionsRevision
+             :model/CardBookmark
+             :model/CollectionPermissionGraphRevision
+             :model/BookmarkOrdering
+             :model/ModelIndexValue
+             :model/PermissionsGroupMembership
+             :model/ViewLog
+             :model/Field
+             :model/QueryCache
+             :model/ApplicationPermissionsRevision
+             :model/LoginHistory
+             :model/Database
+             :model/Session
+             :model/Permissions
+             :model/TaskHistory
+             :model/Setting
+             :model/Activity
+             :model/PulseChannelRecipient
+             :model/TablePrivileges
+             :model/TimelineEvent
+             :model/PersistedInfo
+             :model/HTTPAction
+             :model/QueryExecution
+             :model/DashboardBookmark
+             :model/Table
+             :model/Query
+             :model/PermissionsGroup}
+           (set/difference (set (v2.entity-ids/toucan-models)) (#'v2.entity-ids/entity-id-models))))))

--- a/enterprise/backend/test/metabase_enterprise/serialization/v2/entity_ids_test.clj
+++ b/enterprise/backend/test/metabase_enterprise/serialization/v2/entity_ids_test.clj
@@ -4,6 +4,7 @@
    [clojure.string :as str]
    [clojure.test :refer :all]
    [metabase-enterprise.serialization.v2.entity-ids :as v2.entity-ids]
+   [metabase.config :as config]
    [metabase.models :refer [Collection Dashboard]]
    [metabase.test :as mt]
    [toucan2.core :as t2])
@@ -73,44 +74,46 @@
            if this test fails, check if it really needs it.
            If yes, makes surethe exported data includes `:entity_id` column
            or the model implements [[serdes/hash-fields]] (#35097)"
-    (is (= #{:model/MetricImportantField
-             :model/ModerationReview
-             :model/CollectionBookmark
-             :model/Secret
-             :model/GroupTableAccessPolicy
-             :model/FieldValues
-             :model/ModelIndex
-             :model/DashboardCardSeries
-             :model/ParameterCard
-             :model/QueryAction
-             :model/ImplicitAction
-             :model/User
-             :model/Revision
-             :model/PermissionsRevision
-             :model/CardBookmark
-             :model/CollectionPermissionGraphRevision
-             :model/BookmarkOrdering
-             :model/ModelIndexValue
-             :model/PermissionsGroupMembership
-             :model/ViewLog
-             :model/Field
-             :model/QueryCache
-             :model/ApplicationPermissionsRevision
-             :model/LoginHistory
-             :model/Database
-             :model/Session
-             :model/Permissions
-             :model/TaskHistory
-             :model/Setting
-             :model/Activity
-             :model/PulseChannelRecipient
-             :model/TablePrivileges
-             :model/TimelineEvent
-             :model/PersistedInfo
-             :model/HTTPAction
-             :model/QueryExecution
-             :model/DashboardBookmark
-             :model/Table
-             :model/Query
-             :model/PermissionsGroup}
+    (is (= (cond-> #{:model/MetricImportantField
+                     :model/ModerationReview
+                     :model/CollectionBookmark
+                     :model/Secret
+                     :model/GroupTableAccessPolicy
+                     :model/FieldValues
+                     :model/ModelIndex
+                     :model/DashboardCardSeries
+                     :model/ParameterCard
+                     :model/QueryAction
+                     :model/ImplicitAction
+                     :model/User
+                     :model/Revision
+                     :model/PermissionsRevision
+                     :model/CardBookmark
+                     :model/CollectionPermissionGraphRevision
+                     :model/BookmarkOrdering
+                     :model/ModelIndexValue
+                     :model/PermissionsGroupMembership
+                     :model/ViewLog
+                     :model/Field
+                     :model/QueryCache
+                     :model/ApplicationPermissionsRevision
+                     :model/LoginHistory
+                     :model/Database
+                     :model/Session
+                     :model/Permissions
+                     :model/TaskHistory
+                     :model/Setting
+                     :model/Activity
+                     :model/PulseChannelRecipient
+                     :model/TablePrivileges
+                     :model/TimelineEvent
+                     :model/PersistedInfo
+                     :model/HTTPAction
+                     :model/QueryExecution
+                     :model/DashboardBookmark
+                     :model/Table
+                     :model/Query
+                     :model/PermissionsGroup}
+             config/ee-available?
+             (conj :model/ConnectionImpersonation))
            (set/difference (set (v2.entity-ids/toucan-models)) (#'v2.entity-ids/entity-id-models))))))

